README.md

```diff
@@ -145,6 +133,8 @@ end.
 
    Newer versions of pip are incompatible with some of our requirements, hence
    the pin on the 10.0.1 version.
+   
+   If requirements fail to build, try installing `python3-dev`.
```

Comment:
Please add that this only applies to Ubuntu. I think putting the whole
```
apt install python3-dev
```
command would also be helpful.